x-pack/plugins/lists/common/schemas/request/read_exception_list_item_schema.ts

```diff
@@ -11,11 +11,13 @@ import * as t from 'io-ts';
 import { NamespaceType, id, item_id, namespace_type } from '../common/schemas';
 import { RequiredKeepUndefined } from '../../types';
 
-export const readExceptionListItemSchema = t.partial({
-  id,
-  item_id,
-  namespace_type, // defaults to 'single' if not set during decode
-});
+export const readExceptionListItemSchema = t.exact(
```

Comment:
NOTE: Added, as I believe we're going for adding this to all our schemas?

Reply:
Correct. You have to add `t.exact(` otherwise extra keys and value will be allowed and the exact check will not be capable of working. So anything missing an `t.exact` should be consider a critical bug that we need to fix.

Thank you for this fix.